chia/_tests/blockchain/test_blockchain.py

```diff
@@ -968,23 +1001,21 @@ async def test_invalid_rc_sub_slot_vdf(self, empty_blockchain, bt):
             await _validate_and_add_block(empty_blockchain, block)
 
     @pytest.mark.anyio
-    async def test_genesis_bad_deficit(self, empty_blockchain, bt):
+    async def test_genesis_bad_deficit(self, empty_blockchain: Blockchain, bt: BlockTools) -> None:
         # 2r
         block = bt.get_consecutive_blocks(1, skip_slots=2)[0]
         new_finished_ss = recursive_replace(
             block.finished_sub_slots[-1],
             "reward_chain",
             recursive_replace(
-                block.finished_sub_slots[-1].reward_chain,
-                "deficit",
-                bt.constants.MIN_BLOCKS_PER_CHALLENGE_BLOCK - 1,
+                block.finished_sub_slots[-1].reward_chain, "deficit", bt.constants.MIN_BLOCKS_PER_CHALLENGE_BLOCK - 1
```

Comment:
I think you should avoid making changes unrelated to the purpose of the PR. Especially formatting changes in code that otherwise doesn't change